internal/app/app.go

```diff
@@ -265,9 +254,23 @@ func (a *App) Run(ctx context.Context) error {
 	})
 
 	g.Go(func() error {
-		<-cashInStatusCheckDone
-		<-schedulerStarted
-		<-messageProcessorStarted
+		a.logger.Info("Starting message processor...")
+		close(messageProcessorStarted)
+		a.messageProcessor.Start(gCtx)
```

Comment:
```suggestion
		a.messageProcessor.Start(gCtx)
		close(messageProcessorStarted)
```
Should this be after the msg processor is started?

Reply:
nope, message processor hangs in endless for-loop

Reply:
I didn't want to change more of how things are in this PR, just do this bugfix